virttest/qemu_devices/qcontainer.py

```diff
@@ -2106,6 +2107,8 @@ def define_hbas(qtype, atype, bus, unit, port, qbus, pci_bus, iothread,
                 protocol_cls = qdevices.QBlockdevProtocolFTPS
             elif filename.startswith('ftp:'):
                 protocol_cls = qdevices.QBlockdevProtocolFTP
+            elif filename.startswith('/dev/vhost-vdpa'):
+                protocol_cls = qdevices.QBlockdevProtocolVdpa
```

Comment:
refer to qapi/block-core.json
'virtio-blk-vhost-vdpa':
                     { 'type': 'BlockdevOptionsVirtioBlkVhostVdpa',
                       'if': 'CONFIG_BLKIO' },